lib/bibliothecary/parsers/conda.rb

```diff
@@ -0,0 +1,38 @@
+require 'json'
+
+module Bibliothecary
+  module Parsers
+    class Conda
+      include Bibliothecary::Analyser
+
+      def self.mapping
+        {
+          match_filename("environment.yml") => {
+            kind: 'manifest',
+            parser: :parse_yaml_manifest
+          }
+        }
+      end
+
+      def self.parse_yaml_manifest(file_contents)
+        unparsed_manifest = YAML.load file_contents
+
+        manifest = {}
+        manifest["dependencies"] = unparsed_manifest["dependencies"].map do |dep|
+          # fields seem to be  `name=version=build` or `name` or `name=version`
+          # Handling the `name` case by always appending a ">= 0" to the list, 
+          # and taking first two items.
```

Comment:
haven't seen any online where it's something like 

`name=build`  so not sure if we should handle that. It seems people either hand craft in

`name`
`name=version`
or automate it export being
`name=version=build`